simulation/sub8_simulation/launch/sim_thrusters.launch

```diff
@@ -0,0 +1,67 @@
+<launch>
```

Comment:
This looks like the same thing we have over in the sub8_thruster_mapper. Is there a way to prevent duplicating code?